controls/V-73229.rb

```diff
@@ -1,9 +1,9 @@
 control 'V-73229' do
-  title "Manually managed application account passwords must be at least 15
+  title "Manually managed application account passwords must be at least 14
   characters in length."
   desc "Application/service account passwords must be of sufficient length to
   prevent being easily cracked. Application/service accounts that are manually
-  managed must have passwords at least 15 characters in length."
+  managed must have passwords at least 14 characters in length."
```

Comment:
I would like to update all use of static data values in text and tests to use `inputs`